callbacks/callbacks.py

```diff
@@ -145,13 +208,7 @@ def update_plot(pcode, adm_level):
             config={"displayModeBar": False}, figure=fig_timeseries
         )
         rp_chart = dcc.Graph(config={"displayModeBar": False}, figure=fig_rp)
-        name, exposed_summary = get_summary(df_processed, df_adm, adm_level)
+        name, exposed_summary = get_summary(
+            df_processed, df_adm, adm_level, tercile
+        )
         return exposure_chart, rp_chart, name, exposed_summary
-
-    # TODO: Would be better as a clientside callback, but couldn't seem to get it to work...
-    @app.callback(
-        Output("hover-place-name", "children"), Input("geojson", "hoverData")
-    )
-    def info_hover(feature):
```

Comment:
Aw I thought the hover name thing was cool! But maybe we can bring it back some other way if you think it just needs to be implemented in some other way.

Reply:
Ah ok I'll put it back! I was just cleaning up and thought it was just me that used it.